docs/docusaurus/versioned_docs/version-0.17.23/guides/connecting_to_your_data/components/next_steps.md

```diff
@@ -0,0 +1,5 @@
+Now that you've connected to your data, you'll want to work on these **core skills**:
+
+<!-- - [How to create a Batch of data from an in-memory Spark or Pandas dataframe](/docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe) -->
```

Comment:
@tyler-hoffman is this the link you commented out? looks like the doc only existed in 0.15

Reply:
It did -- this link should have been removed or updated by one of my PRs for the current docs, but at the time that this doc was current the 0.15.x docs were still supported so some links back to them remained for features that didn't have an equivalent any more in 0.17.x.